style.css

```diff
@@ -81,3 +82,194 @@ a {
 .iconDiv > img {
   width: 10%;
 }
+
+/* ====== Created new branch for this styling part ====== */
+
+/* ========= Work section ======= */
+.work {
+  padding: 6rem 0;
```

Comment:
- [ ] Kindly check through your project to ensure that you are using the correct spacing as defined in the figma design